src/features.def

```diff
@@ -68,14 +67,7 @@ THERMOSTAT_IGNORE_NON_VIRTUAL   implies VIRTUAL_SITES_THERMOSTAT
 
 /* DPD features */
 DPD
-TRANS_DPD                       implies DPD
-DPD_MASS_RED                    requires not DPD_MASS_LIN
-DPD_MASS_LIN                    requires not DPD_MASS_RED
-DPD_MASS                        equals DPD_MASS_RED or DPD_MASS_LIN
-DPD_MASS                        implies MASS, DPD
-
-/* Alternative implementation of DPD */
-INTER_DPD
+TUNABLE_SLIP
```

Comment:
Unrelated to DPD

Reply:
Yes, but TUNABLE_SLIP does actually exists, and should be listed there.